cli/internal/controlplane/login.go

```diff
@@ -699,7 +699,7 @@ func readCachedServiceInfo() (*serviceInfo, error) {
 func getServiceMetadata(ctx context.Context, serverUri string) (*model.ServiceMetadata, error) {
 	// Not using a retryable client because when doing `tyger login --local` we first try to use the unix socket
 	// before trying the docker gateway and we don't want to wait for retries.
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/v1/metadata", serverUri), nil)
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/metadata", serverUri), nil)
```

Comment:
The server supports old clients using `/v1/...` but the new client doesn't support old servers, since `/metadata` won't be recognized. I think that is ok, but we should give and error message that makes it clear what is going on instead of just "the server URL does not appear to point to a valid tyger server"